README.rst

```diff
@@ -98,8 +98,8 @@ This module also needs the installation of these other components:
 Top_.
 
 
-Installation
-------------
+Pre-Installation
+----------------
 
 Once you have all prerequisites installed, you must create a DB named cloto in your MySQL server.
 Ensure your mysql path is in your path. If not, you can add executing (change ``/usr/local/`` with your mysql folder):
```

Comment:
I do not understand this sentence right now...

"Ensure your mysql path is in your path. If not, you can add executing (change ``/usr/local/`` with your mysql folder):"